app/src/gui/components/notebook/datagrid_toolbar.tsx

```diff
@@ -38,6 +38,8 @@ interface ToolbarProps {
   handleQueryFunction: any;
 }
 
+const enableFilters = import.meta.env.VITE_ENABLE_RECORD_FILTERS !== 'false';
```

Comment:
Can you handle this in buildconfig with all of the other configuration settings.